src/python/zquantum/core/interfaces/backend_test.py

```diff
@@ -175,3 +374,97 @@ def test_get_bitstring_distribution_wf_simulators(self, wf_simulator):
         assert bitstring_distribution.distribution_dict["111"] == pytest.approx(
             0.5, abs=1e-7
         )
+
+
+class QuantumSimulatorGatesTest:
+    @pytest.mark.parametrize(
+        "initial_gate,tested_gate,target_amplitudes",
+        one_qubit_non_parametric_gates_amplitudes_test_set,
+    )
+    def test_one_qubit_non_parametric_gates_using_amplitudes(
+        self, wf_simulator, initial_gate, tested_gate, target_amplitudes
+    ):
+        # Given
+        qubit_list = [Qubit(0)]
+        gate_1 = Gate(initial_gate, qubits=qubit_list)
+        gate_2 = Gate(tested_gate, qubits=qubit_list)
+
+        circuit = Circuit()
+        circuit.gates = [gate_1, gate_2]
+        import pdb
+
+        pdb.set_trace()
+
```

Comment:
hello my old friend ;)

Reply:
:)

Reply:
Fixed in [bdab82](https://github.com/zapatacomputing/z-quantum-core/pull/145/commits/bdab8249a2b4f5b808b21b0ac7101368b29146a2).